src/Core/Models/SqlTypeConstants.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.DataApiBuilder.Service.Models
+{
+    /// <summary>
+    /// String literal representation of SQL Server data types that are returned by Microsoft.Data.SqlClient.
+    /// </summary>
+    /// <seealso cref="https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/tests/PerformanceTests/Config/Constants.cs"/>
+    public class SqlTypeConstants
```

Comment:
This can be a `static` class as it only has static members right?

```suggestion
    public static class SqlTypeConstants
```